views/retentionReasons.njk

```diff
@@ -0,0 +1,158 @@
+{% extends "./partials/layout.njk" %}
+
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set title = 'Prevent removal of this record' %}
+{% set subtitle = 'Reason for preventing removal' %}
+{% set instructions = 'Select all that may apply. Remove all to stop retaining the record.' %}
+
+{% block content %}
+
+  {{ govukBackLink({
+    text: "Back",
+    href: offenderUrl
+  }) }}
+
+  <div class="govuk-grid-row govuk-body">
+    <div class="govuk-grid-column-full">
+
+      <div class="basic-details">
+        <div class="govuk-!-margin-right-6">
+          <img
+                  src="/app/images/{{ offenderBasics.offenderNo }}/data"
+                  alt="Headshot of {{ offenderBasics.firstName }} {{ offenderBasics.lastName }}"
+                  style="float:left"
+                  height="110"
+                  width="90"
+                  data-qa="offender-image"
+          />
+        </div>
+  
+        <div>
```

Comment:
In order to use the correct font here you'd need to use <div class="govuk-body">govuk-body</div>